tests/sdks/python/containers/localstack_container.py

```
@@ -0,0 +1,89 @@
+        map_file = parser.parse(json_content)
+        client = LocalStackContainer._resolve_client(map_file.config)
+        return client.resolve_secrets(map_file)
+
+    @staticmethod
+    def _resolve_client(
+        config: MapFileConfig,
+    ) -> EnvilderClient:
+        try:
+            provider = SecretProviderFactory.create(config)
+            return EnvilderClient(provider)
+        except Exception:
```

Ruff (0.15.10)

[warning] Do not catch blind exception: `Exception`

(BLE001)